scrapper.py

```diff
@@ -26,16 +31,45 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
+
+    def _link_formatting(link):
+        if 'https://www.kommersant.ru' not in link:
+            link = 'https://www.kommersant.ru' + link
+        return link
 
     def _extract_url(self, article_bs):
-        pass
+        urls_bs = article_bs.findall(class_="uho__link uho__link--overlay")
+        urls_bs_full = [self._link_formatting(url_bs['href']) for url_bs in urls_bs]
+        return urls_bs_full
+
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for url in self.seed_urls:
+            headers = {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:97.0) Gecko/20100101 Firefox/97.0',
+                'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8',
+                'Accept-Encoding': 'gzip, deflate, br'
+                'Accept-Language': 'en-US,en;q=0.5'
+            }
+            for seed_url_index, seed_url in enumerate(self.seed_urls):
+                response = requests.get(url=seed_url)
+            with open(f'{ASSETS_PATH}/{seed_url_index}_seed_url.html', 'w', encoding='utf-8') as file:
```

Comment:
you should not write all articles into ASSETS folder - they will be written later. For now just use only URLS extracted